src/Screens/MainPage/components/SearchMainPage.tsx

```diff
@@ -27,38 +36,54 @@ export const SearchMainPage = () => {
   };
   const handleDropDown = (value: string) => {
     console.log(value);
-
+  };
+  const handleInputBlur = () => {
+    setFocus(false);
+  };
+  const handleInputFocus = () => {
+    setFocus(true);
   };
 
   useEffect(() => {
+    addItem();
+    //call api
+    // addToRecentSearches([],input)
     //change input in store redux
   }, [debouncedValue]);
 
+  useEffect(() => {
+    localStorage.setItem("lastSearches", JSON.stringify(items));
+  }, [items]);
+
+
   useEffect(() => {
     //call api
   }, [Query]);
 
   return (
     <>
-    <FormControl fullWidth>
-    <Container>
-      <Search
-        input={input}
-        searchFunction={handleChange}
-        Icon={() => (
-          <>
-            <SearchIcon />
-          </>
-        )}
-      />
-       <VerticalLine/>
-      <DropDown
-        data={placeholders}
-        onSelect={handleDropDown}
-        placeholder={placeholders[0]}
-      />
-      </Container>
-       </FormControl>
+      <FormControl fullWidth>
+        <Container>
+          <Search
+            onFocus={handleInputFocus}
+            onBlur={handleInputBlur}
+            input={input}
+            searchFunction={handleChange}
+            Icon={() => (
+              <>
+                <SearchIcon />
+              </>
+            )}
+          />
+          <VerticalLine />
+          <DropDown
+            data={placeholders}
+            onSelect={handleDropDown}
+            placeholder={placeholders[0]}
+          />
+        </Container>
+        {focus ? <RecentSearches value={items} setInput={setInput}/> : null}
```

Comment:
if there is no else part in the conditional - you can you like that: 
`{focus && <RecentSearches value={items} setInput={setInput}/> }`